### PR TITLE
Issue 2.1: Added Test cases and methods

### PR DESCRIPTION
**Pull request by ajay567**
**Design**
- Create wiremocks for test scenarios

**Test**
- Added various testing scenarios

### CODE REVIEW

File: src/main/java/org/galatea/starter/entrypoint/IexRestController.java [hunk 1]
```diff
@@ -61,9 +62,23 @@
   @GetMapping(value = "${mvc.iex.getHistoricalPricesPath}", produces = {
       MediaType.APPLICATION_JSON_VALUE})
   public List<IexHistoricalPrices> getHistoricalPrices(
-      @RequestParam(value = "symbol") final String symbols, @RequestParam("range")
-      final String range, @RequestParam(value = "date", required = false) final String date) {
-    return iexService.getHistoricalPricesForSymbols(symbols,range,date);
+      @RequestParam(value = "symbol") final String symbols, @RequestParam(value = "range",
```
Comment (grundygolem2) [inline src/main/java/org/galatea/starter/entrypoint/IexRestController.java]: try to have annotations not wrap lines, it becomes tricky to read

File: src/main/java/org/galatea/starter/entrypoint/IexRestController.java [hunk 2]
```diff
@@ -61,9 +62,23 @@
...
+      required = false) final String range, @RequestParam(value = "date", required = false)
+      final String date) {
+
+    if (symbols == null || symbols.length() == 0) {
```
Comment (grundygolem2) [inline src/main/java/org/galatea/starter/entrypoint/IexRestController.java]: you do null or len=0 3 times. it is probably worth extracting to a utility method

**Review by grundygolem2 — CHANGES_REQUESTED** [review]

File: src/main/java/org/galatea/starter/entrypoint/IexRestController.java (hunk 1 above)
Comment (ajay567) [reply src/main/java/org/galatea/starter/entrypoint/IexRestController.java]: fixed

File: src/main/java/org/galatea/starter/entrypoint/IexRestController.java (hunk 2 above)
Comment (ajay567) [reply src/main/java/org/galatea/starter/entrypoint/IexRestController.java]: method added

**Pull request merged**